src/templates/admin/journal/contact.html

```diff
@@ -42,6 +42,7 @@ <h3>{% trans "Contact" %}</h3>
                                 </select>
                             </div>
                         </div>
+                        <p>{% trans "Required fields are marked with" %} <abbr class="red req" title='{% trans "required" %}'>*</abbr>.</p>
```

Comment:
As we're re-using this it would be good to have a common template that can be used.